magda-web-client/src/Components/Dataset/View/DatasetPageSuggestForm.js

```diff
@@ -83,20 +83,23 @@ export default class DatasetPageSuggestForm extends React.Component {
                 maxHeight: "95vh"
             }
         };
+        var contactPointMatch = this.props.contactPoint.match(
```

Comment:
1> better use `let` or `const` rather than var --- I think it (`var`) might not pass our CI code formating check
2> as `this.props.contactPoint` potentially could be a null or undefined value (that has no `.match`), to avoid throw an error, we probably want check its value first.
e.g. let contactPointMatch = ""; if(typeof this.props?.contactPoint === "string") {....}
3> And I don't think `match` will return a string type? it should return an array
4> `"sdds dssd@dsd.com.au sdsd".match(/([a-zA-Z0-9.-]+@[a-zA-Z0-9.-]+.[a-zA-Z0-9._-]+)/g)` will match
`"dssd@dsd.com.au sdsd"`  --- we probably need a better email regex here

Reply:
1) fixed (although var seemed to pass the tests with the previous commit)
2) Did an initial check
3) Also fixed
4) I struggled to find a better email regex, the short-term solution I found was just getting the first email match (should be sufficient as there is only one publisher), and just split by whitespace. If you have other regex recommendations let me know.